pkg/northbound/gnmi/set.go

```diff
@@ -187,15 +188,19 @@ func (s *Server) Set(ctx context.Context, req *gnmi.SetRequest) (*gnmi.SetRespon
 		return nil, status.Error(codes.Internal, errSet.Error())
 	}
 	if userName != "" {
+		s.mu.Lock()
```

Comment:
This lock is not necessary since there’s no volatile state being updated and the Atomix stores are already thread safe.